src/tests/example.spec.js

```diff
@@ -24,4 +25,105 @@ test.describe('', () => {
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+        await expect(inventoryPage.headerTitle).toBeVisible();
+
+        await inventoryPage.sortByValue('lohi');
+        const inventoryItemsLoHi = await inventoryPage.inventoryItems.allTextContents();
+        await inventoryPage.sortByValue('hilo');
+        const inventoryItemsHiLo = await inventoryPage.inventoryItems.allTextContents();
+        expect(inventoryItemsLoHi).not.toEqual(inventoryItemsHiLo);
+    });
+
+    test('Add several random products', async ({ loginPage, inventoryPage, shopingCartPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+        await expect(inventoryPage.headerTitle).toBeVisible();
+
+        const itemsCount = await inventoryPage.inventoryItems.count();
+        const id1 = getRandomProductId(itemsCount);
+        const id2 = getRandomProductId(itemsCount);
+
+        const id1Info = await inventoryPage.getProductInfoById(id1);
+        const id2Info = await inventoryPage.getProductInfoById(id2);
+
+        await inventoryPage.addItemToCartById(id1);
+        await inventoryPage.addItemToCartById(id2);
+
+        await inventoryPage.shopingCart.click();
+        expect(await shopingCartPage.cartItems.count()).toEqual(2);
+
+        const prod1Desc = await shopingCartPage.getProductDescByName(id1Info.productName);
+        const prod2Desc = await shopingCartPage.getProductDescByName(id2Info.productName);
+
+        const prod1Price = await shopingCartPage.getProductPriceByName(id1Info.productName);
+        const prod2Price = await shopingCartPage.getProductPriceByName(id2Info.productName);
+
+        expect(prod1Desc).toEqual(id1Info.productDesc);
+        expect(prod2Desc).toEqual(id2Info.productDesc);
+
+        expect(prod1Price).toEqual(id1Info.productPrice);
+        expect(prod2Price).toEqual(id2Info.productPrice);
+    });
+
+    test('Verify checkout', async ({
+        loginPage, inventoryPage, shopingCartPage, yourInformationPage, overviewPage,
+    }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+        await expect(inventoryPage.headerTitle).toBeVisible();
+
+        const itemsCount = await inventoryPage.inventoryItems.count();
+        const id1 = getRandomProductId(itemsCount);
+        const id2 = getRandomProductId(itemsCount);
+
+        const id1Info = await inventoryPage.getProductInfoById(id1);
+        const id2Info = await inventoryPage.getProductInfoById(id2);
+
+        await inventoryPage.addItemToCartById(id1);
+        await inventoryPage.addItemToCartById(id2);
+
+        await inventoryPage.shopingCart.click();
+        await shopingCartPage.clickCheckout();
+
+        await yourInformationPage.fillFirstName('Notashenka');
+        await yourInformationPage.fillLastName('Popeliushenka');
+        await yourInformationPage.fillPostalCode('58000');
```

Comment:
We can avoid hardcoding of the user here and create an object with user data and store it here or in a separate module